src/components/ListItem.tsx

```diff
@@ -138,17 +146,19 @@ const ListItem = ({
 
   return (
     <Container
+      disabled={disabled || !onPress}
       onPressIn={onPressIn}
       onPressOut={onPressOut}
       onLongPress={() => setPressed(true)}
       pressed={pressed}
-      delayLongPress={200}>
+      delayLongPress={200}
+      testID='list-item'>
       {iconToRender}
       <FlexContainer>
         {titleToRender}
         <DescriptionContainer>
           {badgeLabel && <BadgeLabel pressed={pressed}>{badgeLabel}</BadgeLabel>}
```

Comment:
```suggestion
           {badgeLabel && badgeLabel.length > 0 && <BadgeLabel pressed={pressed}>{badgeLabel}</BadgeLabel>}
```
optional since you didn't change that line